src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertyAccessorSymbol.cs

```diff
@@ -525,6 +525,14 @@ internal override bool GenerateDebugInfo
             }
         }
 
+        public sealed override bool AreLocalsZeroed
+        {
+            get
+            {
+                return _property.AreLocalsZeroed && base.AreLocalsZeroed;
```

Comment:
>return _property.AreLocalsZeroed && base.AreLocalsZeroed; [](start = 16, length = 57)

I think we should also handle a case when the attribute is explicitly applied to an accessor. Please add a test for a scenario like that.